src/Filters.jl

```diff
@@ -182,6 +182,7 @@ struct Shuffle <: Filter
     element_size::UInt32
 end
 Shuffle() = Shuffle(0)
+filtername(::Type{Shuffle}) = ""
```

Comment:
This definition would not be used.
Filters with an id `<=255` do not have a filtername by definition of the h5 format spec.